clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -769,6 +769,9 @@ public boolean refreshCommittedOffsetsIfNeeded(Timer timer) {
                     this.subscriptions.seekUnvalidated(tp, position);
 
                     log.info("Setting offset for partition {} to the committed offset {}", tp, position);
+                } else {
+                    log.info("Ignoring the returned {} since its partition {} is no longer assigned",
```

Comment:
nit: I'd suggest "Ignoring the fetched committed offset"

Reply:
I originally did that, but then I realize `OffsetAndMetadata#toString` contains this already so I decided to avoid duplicated wording.